unit/test_connect_to_membership.py

```diff
@@ -4,9 +4,15 @@
 from library.cartridge_connect_to_membership import connect_to_membership
 
 
-def call_probe_instance(console_sock, hostvars, play_hosts=None):
+def call_probe_instance(console_sock, role_vars, play_hosts=None):
     if play_hosts is None:
-        play_hosts = hostvars.keys()
+        play_hosts = role_vars.keys()
+
+    hostvars = {}
+    for instance_name, instance_role_vars in role_vars.items():
+        hostvars[instance_name] = {
+            'role_vars': instance_role_vars
```

Comment:
Why modules don't accept `role_vars` directly instead of `hostvars`?

Reply:
Because `role_vars` - vars on one host, `hostvars` - vars on all hosts